acceptance/acceptance_test.go

```diff
@@ -411,6 +411,10 @@ func getSkipReason(config *internal.TestConfig, configPath string) string {
 		return "Disabled via SkipLocal setting in " + configPath
 	}
 
+	if isTruePtr(config.SkipOnDbr) && WorkspaceTmpDir {
```

Comment:
question - is intentionally placed before Forcerun check?

Reply:
Thanks, no it's not. Fixed in https://github.com/databricks/cli/pull/3552